presto-docs/src/main/sphinx/release/release-0.244.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.244
+=============
+
+**Highlights**
+==============
+* Improve performance of NestedLoopJoinOperator.
+* Add support for spilling joins when using materialized exchanges.
+
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix an issue where queries could encounter an int overflow error during spilling.
+* Fix hamming_distance to handle code point zero correctly.
```

Comment:
Should be
```
:func:`hamming_distance`
```
as according to the [Release Notes Guideline](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines)